docs/DeveloperGuide.md

```diff
@@ -362,32 +362,58 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Use case: UC4 - Delete Fosterers**
 
-**Preconditions:** A list of fosterers is currently displayed.
-
 **MSS**
 
-1. Foster Manager requests to delete fosterers referenced by their index shown in the list.
-2. System deletes selected fosterers.
-3. System displays the updated list of fosterers.
+1. Foster Manager requests to view a list of fosterers.
+2. Foster Manager requests to delete fosterers referenced by their index shown in the list.
+3. System deletes selected fosterers.
+4. System displays the updated list of fosterers.
 
     Use case ends.
 
 **Extensions**
 
-* 1a. The displayed list is empty. 
+* 2a. The displayed list is empty. 
```

Comment:
I think they can still request to list, just that there will be 0 persons listed. only when they request to delete a fosterer, then the system will detect the invalid index (since the list size = 0)

Reply:
Is it better to rephrase it so it's clearer? Smth like "The displayed list is empty, so any indices indicated will be invalid" or smth less wordy HAHA